### PR TITLE
Adding a web page

### LINKED ISSUE #10: Adding a web page
Adding CSS and HTML files of a static web page showing the name of our project and a short description about it.

### PR DESCRIPTION
**Pull request by OmerCS8**
-Adding CSS and HTML files of a static web page showing the name of our project and a short description about it.

- Setup Vagrant to automatically bring our application up as well as
make it accessible from a web browser
After running `vagrant up` the application can be accessed at:
> http://127.0.0.1:8000/

fix #10

### CODE REVIEW

**Review by MatanP12 — APPROVED** [review]
LGTM

**Comment by natifridman** [comment]
@OmerCS8 Please take a look at the first challenge about how to create a view and template. I think the homepage should be a template and not an app.
Also, maybe it will be better to move the static folder to the main folder.

**Review by AlonShakaroff — APPROVED** [review]
Looks really good, good job

File: homepage/templates/homepage/homepage.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                <LI>Omer Cohen Shor</LI>
+                <LI>Yuli Suliman</LI>
+                <LI>Alon Shakaroff</LI>
+                <LI>Matan Peretz</LI>
+                <LI>Tomer Newman</LI>
+            </UL>
+        </div>
+    </main>
+</body>
+</html>
```
Comment (TomerNewmanPrograms) [inline homepage/templates/homepage/homepage.html:57]: In the end of the file, is there a need for a newline (homepage.html)?
Everything else looks great

Comment (abiton1) [reply homepage/templates/homepage/homepage.html:57]: Yes, please add a new line at the end of the file

**Pull request merged**